Packs/Stairwell/Integrations/Inception/Inception.py

```diff
@@ -40,8 +40,10 @@ def file_enrichment_command(client, file_hash):
             responseJson = json.loads(response)
```

Comment:
Why are using json.dumps in line 38 and then then json.loads in line 40? it does not have any effect. If it is to check that "attributes" is in the response string use:
`response_json = client.get_file_reputation(file_hash)`
`response_data = response_json.get('data', {})`
`if "attributes" in response_data:`
 `    ...`